great_expectations/core/batch_manager.py

```diff
@@ -98,7 +98,7 @@ def active_batch_id(self) -> Optional[str]:
         the list of "Batch" objects, each of whose BatchData has already been loaded (and cached).  Since BatchData IDs
         are from the same name space as Batch IDs, this helps avoid unnecessary loading of data from different backends.
         """
-        active_batch_data_id: Optional[str] = self._active_batch_data_id
+        active_batch_data_id: Optional[str] = self.active_batch_data_id
```

Comment:
Was this a bug? What is the difference between these two attributes?

Reply:
@NathanFarmer Using full property takes advantage of the checks that property method performs -- this is more robust.